README.md

```diff
@@ -258,6 +259,106 @@ qupid shuffle \
     --output matches.tsv
 ```
 
-## Help with qupid
+## QIIME 2 Usage
 
-If you encounter a bug in qupid, please post a GitHub issue and we will get to it as soon as we can. We welcome any ideas or documentation updates/fixes so please submit an issue and/or a pull request if you have thoughts on making qupid better.
+Qupid provides support for the popular QIIME 2 framework of microbiome data analysis.
+We assume in this tutorial that you are familiar with using QIIME 2 on the command line.
+If not, we recommend you read the excellent [documentation](https://docs.qiime2.org/) before you get started with Qupid.
+
+Run `qiime qupid --help` to see all possible commands.
+
```

Comment:
If this is all auto generated, maybe best to just leave it as Run qiime qupid --help, rather than trying to keep it in sync as code is modified

Reply:
Yeah that makes sense.